BUILD.md

```diff
@@ -169,9 +169,11 @@ traffic to the docker-compose environment:
 kubectl version
 
 # simulate traffic
-bin/go-run controller/script/simulate-proxy --kubeconfig ~/.kube/config --addr $DOCKER_IP:8086 --max-pods 10 --sleep 10ms
+bin/go-run controller/script/simulate-proxy --kubeconfig ~/.kube/config --max-pods 10 --sleep 10ms
 ```
-
+Note that the Kubernetes cluster your system is configured to talk to must not be referenced via 
```

Comment:
What do you think about breaking the dependency on a real Kubernetes cluster by randomly generating namespace names, pod names, and deployment names?  Or making them configurable (e.g. please generate metrics for deployment "foo" in namespace "bar")?